### PR TITLE
Fix random number generation on ESP8266

### PR DESCRIPTION
**Pull request by rerobika**
This patch uses the onboard RTC for generating random seed. It also improves the print handler to support float values.
The rom segment is slightly increased to fit to the latest master.

JerryScript-DCO-1.0-Signed-off-by: Robert Fancsik frobert@inf.u-szeged.hu

### CODE REVIEW

File: targets/esp8266/user/jerry_extapi.c [hunk 1]
```diff
@@ -73,9 +73,20 @@ DELCARE_HANDLER(print) {
...
         free (buffer);
       }
-      else
+      else if (jerry_value_is_number (args_p[cc]))
       {
-        printf ("(%d) ", args_p[cc]);
+        double number = jerry_get_number_value (args_p[cc]);
+        if ((int) number == number)
+        {
+          printf ("%d", number);
```
Comment (robertsipka) [inline targets/esp8266/user/jerry_extapi.c]: Does not this cause a warning here? Is it prints a correct number at all? Not just rubbish?

Comment (rerobika) [reply targets/esp8266/user/jerry_extapi.c]: I tested it, and it caused an issue. Thanks for your suggestion, I'm going to fix it.

Comment (rerobika) [reply targets/esp8266/user/jerry_extapi.c]: Fixed, now it works as intended.

**Comment by robertsipka** [comment]
LGTM (informally)

**Comment by yichoi** [comment]
LGTM

**Pull request merged**